index.html

```diff
@@ -23,24 +25,37 @@
       </nav>
 
 
-      <section>
+      <section class="sub">
         <h1>Startrly</h1>
         <sub>A catchy tagline for a great solution</sub>
-        <a href="#">Some Call to Action</a>
+        <br>
+        <br>
+        <p></p>
+        <a href="#"></a>
+        <input name="name" type="text" placeholder="    Some Call To Action">
+      </div>
```

Comment:
Changes to in HTML to make box around link -> can be accomplished using border in css with margins, but liked that you applied what we learned yesterday, though not sure if you meant for viewer of website to be able to type into the box?